### PR TITLE
Add ASCII art for Mac

### PR DESCRIPTION
**Pull request by terrajobst**
@KrzysztofCwalina @stephentoub @richlander: Like?

### CODE REVIEW

File: demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs [hunk 1]
```diff
@@ -145,8 +145,43 @@ @@                   @@@
...
+   @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+    @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+     @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+      @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+       @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+         @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+          :@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@;
+             @@@@@@@;           ;@@@@@@@";
+
+        Console.ForegroundColor = ConsoleColor.White;
```
Comment (stephentoub) [inline demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs]: Consider setting both ForegroundColor and BackgroundColor here to ConsoleColor.Red or White or something similar.

Comment (stephentoub) [inline demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs]: Consider also setting BackgroundColor to White.  That'll hide the '@' and give it a feel like what's there for both Windows and Linux.

**Comment by davidfowl** [comment]
How do I run the demo on OS X?

**Comment by stephentoub** [comment]
Patience, my dear David :)

**Comment by stephentoub** [comment]
LGTM

File: demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs (hunk 1 above)
Comment (terrajobst) [inline demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs]: I tried that but it looked quite bad :-(

**Pull request merged**